packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -282,6 +282,10 @@ export const messages = {
   org_select_text: 'Select an org to set as default',
   org_expired: 'Expired',
   missing_default_org: 'No Default Org Set',
+  invalid_default_org:
+    'The org you’ve chosen as your default org isn’t valid. Please choose a different one.',
```

Comment:
what is an invalid org ?

Reply:
I think of an "invalid org" as one that isn't able to be used.  So it would be either an expired or deleted org.

Reply:
Thanks Daphne. why are we differentiation this and the following use case and giving different error message to the user?

Reply:
An invalid org is something you type in the config file that is not an org you have in the cli

Reply:
thanks Cristi. It seems weird that we show an invalid org in the org picker. Shouldn't we even let that to happen?

Reply:
We will not show it there, we will show the message. There's a screenshot in Daphne's test doc, like the last one in the pr description (that was before sonal's text review)